docs/adr/0027-cicd-system.md

```diff
@@ -0,0 +1,116 @@
+# Use GitHub Actions for CI/CD
+
+**User Story:** [ES 416](https://jiraent.cms.gov/browse/ES-416)
+
+The on-premise CircleCI Enterprise offering that we currently use will be
+deprecated at some point in the future and we need to identify which CI/CD
+system we will migrate to.
+
+Decision drivers:
+
+* FedRAMP compliance
+* Security
+* Functionality
+* Developer friendliness
+* Reliability
+* Maintenance cost
+
+## Considered Alternatives
+
+* GitHub Actions (GitHub-hosted runner)
+* GitHub Actions (self-hosted runner)
+* SaaS CircleCI
+* AWS CodePipeline + CodeBuild (managed)
+* CloudBees Core
+
+## Decision Outcome
+
+* Chosen Alternative: GitHub Actions (GitHub-hosted runner)
+
+## Pros and Cons of the Alternatives <!-- optional -->
+
+### GitHub Actions (GitHub-hosted runner)
+
+* `+` Integration with GitHub, which we already use as our version control
+  system.
+* `+` Developer friendly: Workflow results are visible directly in the repo. No
+  need to log into another system.
+* `+` Managed: it requires zero operational overhead to maintain the servers,
+  keep them patched, perform system maintenance, and other activities related to
+  infrastructure management.
+* `+` Highly configurable, rich feature set
+* `+` Modularized configurations: Wide array of shared code and tooling
+  available in the GitHub Actions marketplace
+* `-` Moderate learning curve: Truss has been increasingly using GitHub Actions
+  on other client projects but there will still be a learning curve and
+  moderate level of effort to convert this project's CircleCI configuration to
+  GitHub Actions.
+* `-` It locks us into using GitHub. If we end up wanting to use some other git
+  hosting solution, we will need to rewrite all of our CI configurations and
+  tooling.
+* `-` GitHub Actions was released for general availability in November 2019, so
+  it is still a very young offering. This likely will result in rough edges that
+  won't be present in more mature CI offerings.
+
+### GitHub Actions (self-hosted runner)
+
+* `+` Most of the same pros for GitHub-hosted runners apply here
+* `+` More control over the runner's hardware and software configuration
+* `-` EASi's application code is hosted in a public repository, and GitHub
+  recommends only using self-hosted runners with private repositories. See
+  [Self-hosted runner security with public
+  repositories](https://docs.github.com/en/actions/hosting-your-own-runners/about-self-hosted-runners#self-hosted-runner-security-with-public-repositories).
+* `-` Higher maintenance cost than GitHub-hosted runners as it requires
+  deploying and maintaining the runner infrastructure
+
+### SaaS CircleCI
```

Comment:
It might be worth calling out that this option also doesn't need to be accessed over the VPN.